### ISSUE TITLE
SVM Runtime Error

### ISSUE THREAD

**Issue by raoberman** [IntelPython/daal4py#124]
I found a strange error while running SVM.  When I run my data (a sklearn toy dataset) through the SVM training algorithm, I receive the following error:

![svmerrors](https://user-images.githubusercontent.com/26121125/61478629-bb7f5580-a946-11e9-9a40-d3253b0b5385.PNG)

However, both the data and the labels arguments put into the function have the same number of rows and are in an acceptable daal4py format, so I am unsure why I am receiving this error (see below).  

![shapetrain](https://user-images.githubusercontent.com/26121125/61479426-a99eb200-a948-11e9-9fe4-64a1bc5e85c9.PNG)

This error occurs when I run the data with numpy arrays and Pandas DataFrames.

When I run the same data set in the same format (numpy arrays) through KNN Classification, the error does not occur and the code runs fine.

My code that causes the error:

[svmerrors.zip](https://github.com/IntelPython/daal4py/files/3407742/svmerrors.zip)

A successful run will print the statement "Example run successful!"

**Comment by Alexander-Makaryev** [comment]
Hello,
DAAL expects that labels in SVM will be -1 and 1.
https://software.intel.com/en-us/daal-programming-guide-details-32
In your example they are 0 and 1.
I am totally agree that the text of error is confusing.
The simple workaround can be in replacement values in input array of labels:
y_train = np.where(y_train == 0, -1, y_train)
or using of daal4py.sklearn.svm with implementation of the same behavior
https://github.com/IntelPython/daal4py/blob/master/daal4py/sklearn/svm/svm.py#L157

**Comment by nathan-greeneltch-intel** [comment]
@Alexander-Makaryev This makes alot of sense. Can we add a more helpful error message? and make sure this gets called out in the docs?

**Comment by Alexander-Makaryev** [comment]
@kiselik @SmirnovEgirRu
I will add DAAL team members.
Looks like this error message daal4py gets from DAAL.

**Comment by kiselik** [comment]
Hi all,

the issue is under analyzing next 2 weeks. I’ll notify you when any results have achieved.

**Comment by mayujsw** [comment]
may i know when it will be closed

**Comment by PetrovKP** [comment]
We will definitely add support any values (include 0/1) for binary labels in the near future on side of DAAL.

**Comment by PetrovKP** [comment]
We have fix this problem from DAAL2020.2. This release is now available in conda. https://anaconda.org/intel/daal
